8.x/core-concepts/annotations.md

```diff
@@ -29,19 +29,19 @@ class Main {
 ```
 **OUTPUT**
 
-```bash{1}
+```bash
 I am a rose
 ```
 
-In the above example, both the superclass and subclass include the method displayInfo(). However the method of the subclass is called during the program execution.
+In the above example, both the superclass and subclass include the method `displayInfo()`. However the method of the subclass is called during the program execution due to the `@Override` annotation.
 
-## Annotations in php language
+## Annotations in PHP language
 
-Annotations are called `DocBlock` in the php language. A DocBlock is a piece of documentation in the source code that provides information about a specific class, method or other structural elements.*<sup>[2](#2)</sup>*
+There were no annotations in the PHP language before version 8.0, so the metadata was written into the `DocBlock`. A DocBlock is a piece of documentation in the source code that provides information about a specific class, method or other structural elements.*<sup>[2](#2)</sup>*
```

Comment:
according to the used tag above this section, you are supposed to explain annotations in PHP.(defining doc block is fine by the way, don't delete it) I recommend you to even compare the annotations and comments in PHP as a brief. because as a beginner mind, i was confused about the differences. It may help a lot

Reply:
@reyhane-rangamiz : Hi. Thanks for your review. As described in the video, the larammerce platform is using DockBlock, not the annotations of php 8.0. So I think the larammerce documentation must focus on the DockBlock concept which is explained in reference #2 .